src/Analysis/Ast/Impl/Specializations/BuiltinsSpecializations.cs

```diff
@@ -91,6 +92,50 @@ public static IMember List(IPythonInterpreter interpreter, IPythonFunctionOverlo
             return args.Count > 0 && args[0] is PythonCollection c ? c.Contents.FirstOrDefault() : null;
         }
 
+        public static IMember Super(IPythonModule declaringModule, IPythonFunctionOverload overload, IArgumentSet argSet, IndexSpan indexSpan) {
+            if (argSet.Arguments.Any()) {
+                //multi argument form can have 1 or 2 arguments
+                var classType = argSet.Argument<IMember>(0).GetPythonType<IPythonClassType>();
+                var objOrType = argSet.Argument<IMember>(1).GetPythonType<IPythonClassType>();
```

Comment:
objOrType , I think I need to try to cast to a python instance too.  IPythonInstance ?

Reply:
`GetPythonType` handles instances. It is an extension to `IMember` which includes instances, variables and types.

Reply:
updated.. a bit more complex to figure out isinstance or issubclass.. should they be helper functions?